host/tests/L3_TestCases/dsVideoDevice/dsVideoDevice_test1_FrameratePreChangePostChangeCallback_Verify.py

```diff
@@ -94,30 +95,30 @@ def testFunction(self):
         for device in SupportedDevices:
             self.testdsVideoDevice.setFRFMode(device, 'Enable')
 
-            for streamUrl in self.StreamUrl:
-                streamPath = self.testDownloadSingleStream(streamUrl)
-                self.testPlayer.play(streamPath)
+            for streamUrl in self.testStreams:
+                streamUrl = streamUrl.replace("\\", "/")
+                self.testPlayer.play(streamUrl)
                 time.sleep(5)
 
-                self.log.stepStart(f'Check Frame Rate Pre Post Change Callback device:{device}, Stream:{os.path.basename(streamPath)}')
+                self.log.stepStart(f'Check Frame Rate Pre Post Change Callback device:{device}, Stream:{os.path.basename(streamUrl)}')
                 frameRateChangeTime = self.testdsVideoDevice.getFrameratePrePostChangeCallbackStatus()
 
                 if("Pre" in frameRateChangeTime):
                     preTime = frameRateChangeTime["Pre"]
                     if("Post" in frameRateChangeTime):
                         postTime = frameRateChangeTime["Post"]
-                        self.log.stepResult(True, f'Check Frame Rate Pre Post Change Callback device:{device}, Stream:{os.path.basename(streamPath)}')
+                        self.log.stepResult(True, f'Check Frame Rate Pre Post Change Callback device:{device}, Stream:{os.path.basename(streamUrl)}')
                         self.log.step(f'Time Difference between Pre and Post FrameRate change: {postTime - preTime}')
                     else:
                         self.log.error("Post Framerate change callback not triggered")
-                        self.log.stepResult(False, f'Check Frame Rate Pre Post Change Callback device:{device}, Stream:{os.path.basename(streamPath)}')
+                        self.log.stepResult(False, f'Check Frame Rate Pre Post Change Callback device:{device}, Stream:{os.path.basename(streamUrl)}')
                 else:
                     self.log.error("Pre Framerate change callback not triggered")
-                    self.log.stepResult(False, f'Check Frame Rate Pre Post Change Callback device:{device}, Stream:{os.path.basename(streamPath)}')
+                    self.log.stepResult(False, f'Check Frame Rate Pre Post Change Callback device:{device}, Stream:{os.path.basename(streamUrl)}')
 
                 self.testPlayer.stop()
 
-                self.testDeleteSingleStream(streamPath)
+                self.testDeleteSingleStream(streamUrl)
```

Comment:
Updated the the test cases with the required change.

Looks like this change will not be applicable to this test "dsVideoDevice_test3_SetDisplayFramerate.py", because here for every framerate all videos are playing. So, not included here.

Reply:
Updated the same in dsVideoDevice_test3_SetDisplayFramerate.py as well